### ISSUE TITLE
Add data migration for breweries

### ISSUE THREAD

**Issue by Johnnyk737** [openbrewerydb/openbrewerydb-java-api#17]
Add data migration from openbrewerydb/breweries.csv

**Comment by noebrito** [comment]
Just realized that the data set has addresses instead of a geolocation?

**Comment by Johnnyk737** [comment]
Yeah, it has the lat/long values, but it does also have full addresses. Also, website URL. Here are the list of things.

id - exist
obdb_id
name - exist
brewery_type - exist
street
address_2
address_3
city - exist
state - exist
county_province
postal_code
website_url
phone
created_at - exist
updated_at - exist
country - exist
longitude - exist
latitude - exist
tags

**Comment by noebrito** [comment]
Can you link me to the dataset? I must be looking in the wrong place.

**Comment by Johnnyk737** [comment]
https://github.com/openbrewerydb/openbrewerydb. It's the breweries file with csv, json, or sql types.

**Comment by noebrito** [comment]
Oh I see. I was looking under the data directory.
